package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keycloak/keycloak-admin-client",
-  "version": "18.0.0-dev.1",
+  "version": "999-dev",
```

Comment:
I don't think this is a valid [SemVer](https://semver.org/).

Reply:
This PR was just generated by keycloak-release, and since we bumped to 999-SNAPSHOT in Maven's that's what it ended up with here.

I don't think this is the correct either, but not sure what is best practices in npm with regards to "dev/snapshot" versions. @jonkoops I think you said not even having the "versions" in main?